js/main.js

```diff
@@ -0,0 +1,115 @@
+
+//The book class
+class Book {
+  constructor(title, author, pages, status, id) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.status = status;
+  }
+
+  togglestat() {
```

Comment:
change the name of the function to a `camelCase` one or just name it `toggle`

Reply:
Okey, updated